config.py

```diff
@@ -3,51 +3,68 @@
 Config file
 ===========
 
-Configuration parameters for the study.
+Set the configuration parameters for the study.
 """
 
 import os
 from collections import defaultdict
+from bids import BIDSLayout
 import numpy as np
-from mne.datasets import sample
 
 # ``plot``  : boolean
 #   If True, the scripts will generate plots.
 #   If running the scripts from a notebook or spyder
 #   run %matplotlib qt in the command line to get the plots in extra windows
 
+plot = True
 plot = False
 
+
+# ``kind`` : str
+#   The kind of data to be analyzed, can be 'eeg', or 'meg'.
+
+# kind = 'eeg'
+kind = 'meg'
+
+# BIDS params
+# see: bids-specification.rtfd.io/en/latest/99-appendices/04-entity-table.html
+ses = None
+# task = 'matchingpennies'
+task = 'somato'
+run = None
```

Comment:
I spent a good 15 minutes debugging why it wasn't working for my data only to find out that you overwrite this variable down below with `run = None`. You should really infer all this from the structure as much as possible.